yt/data_objects/static_output.py

```diff
@@ -326,6 +327,8 @@ def periodicity(self):
         elif getattr(self, "_domain_override", False):
             # dataset loaded with a bounding box
             return (False, False, False)
+        if self._set_periodicity is not None:
+            return self._set_periodicity
         return self._periodicity
```

Comment:
nit; personally I find this better but don't commit it if you find it confusing
```suggestion
        return self._periodicity_override or self._periodicity
```

Reply:
I do think this is a little confusing so I would personally leave it as is for the moment.

Reply:
... at least for myself

Reply:
That's fine. In that case I would suggest this instead:
```suggestion
        if self._set_periodicity is not None:
            return self._set_periodicity
        else:
            return self._periodicity
```
It's exactly equivalent but I have a small preference for aligned return statements

Reply:
Yep, I prefer that style too!